src/main/java/forestry/apiculture/ModuleApiculture.java

```diff
@@ -988,6 +990,39 @@ public boolean processIMCMessage(IMCMessage message) {
 				HiveConfig.addBlacklistedDim(dim);
 			}
 			return true;
+		} else if (message.key.equals("add-plantable-flower")) {
+			try {
```

Comment:
This method is getting very long, can you separate out these new cases into new methods?